stylelint-polaris/index.js

```diff
@@ -371,6 +371,7 @@ const stylelintPolarisCoverageOptions = {
     },
   ],
   conventions: {
+    'selector-disallowed-list': [/class[*^]='Polaris-[a-z_-]+'/gi],
```

Comment:
```suggestion
    'selector-disallowed-list': [/class[*^~]?='Polaris-[a-z_-]+'/gi],
```

Found a couple more permutation in web

Reply:
Thanks Aaron!!!